framework/src/variables/MooseVariableFV.C

```diff
@@ -532,12 +525,7 @@ MooseVariableFV<OutputType>::getNeighborValue(const Elem * const neighbor,
   if (neighbor && this->hasBlocks(neighbor->subdomain_id()))
     return getElemValue(neighbor);
   else
-    // If we don't have a neighbor, then we're along a boundary
-    // Linear interpolation: face_value = (elem_value + neighbor_value) / 2. Note that weights of
-    // 1/2 are perfectly appropriate here because we can arbitrarily put our ghost cell centroid
-    // anywhere and by convention we locate it such that a line drawn between the ghost cell
-    // centroid and the element centroid is perfectly bisected by the face centroid
-    return 2. * getBoundaryFaceValue(fi) - elem_value;
+    return getBoundaryFaceValue(fi);
```

Comment:
I feel like an error here would be better in the spirit of this PR?

Reply:
I deleted this function in general, no need to wrap `elemValue` anymore.